testing/bloaty/APKBUILD

```diff
@@ -0,0 +1,41 @@
+#-*-mode: Shell-script; coding: utf-8;-*-
+# Maintainer: Mitch Tishmack <mitch.tishmack@gmail.com>
+_commitid=2234386bcee7297dfa1b6d8a5d20f95ea4ed9bb0
+pkgname=bloaty
+# Fake a version until upstream does one
+# format is simply 0.0.0.0.${commit_count}
+pkgver=0.0.0.0.79 
```

Comment:
For the date should it be the last commit or just say today? Or is it more whatever works in this case, I'm impartial really, my goal with that version was to make a silly version that was almost guaranteed to be lower than whatever was chosen by the upstream.